src/components/SocialMediaIcons.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+import type { IconType } from "react-icons";
+import {
+  FaFacebook,
+  FaGithub,
+  FaInstagram,
+  FaLinkedin,
+  FaTiktok,
+  FaYoutube,
+} from "react-icons/fa";
+
+const ICON_SIZE = 30;
+
+const SocialMediaIcons = ({ classes }: { classes: string }) => {
+  type socialMediaType = {
+    link: string;
+    icon: IconType;
+  };
+
+  const links: socialMediaType[] = [
+    {
+      link: "https://www.facebook.com/CodeTheChangeYYC/",
+      icon: FaFacebook,
+    },
+    {
+      link: "https://www.instagram.com/codethechangeyyc/",
+      icon: FaInstagram,
+    },
+    {
+      link: "https://www.tiktok.com/@codethechangeyyc/",
+      icon: FaTiktok,
+    },
+    {
+      link: "https://www.linkedin.com/company/code-the-change-yyc/",
+      icon: FaLinkedin,
+    },
+    {
+      link: "https://www.youtube.com/channel/UC4wZt-bCL31HjxUF-zc5U_g",
+      icon: FaYoutube,
+    },
+    {
+      link: "https://github.com/Code-the-Change-YYC",
+      icon: FaGithub,
+    },
+  ];
+
+  return (
+    <div className={`flex items-start space-x-6 pt-5 ${classes}`}>
```

Comment:
Do you think we could put white right here and remove the `classes` prop? If we use it somewhere else and we require this flexibility, we should use tailwind merge because concatenating classnames may have unexpected behaviour.

Reply:
I agree, thanks for the suggestion.